launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -540,3 +593,17 @@ void CSettingsView::on_spinBoxReservedArea_valueChanged(int arg1)
 	node->Float() = float(arg1) / 100; // percentage -> ratio
 }
 
+void CSettingsView::addItemsToControls()
+{
+	ui->comboBoxAutosaveMode->addItem("Off");
```

Comment:
Enable translations for those strings using tr()
```suggestion
	ui->comboBoxAutosaveMode->addItem(tr("Off"));
```

Reply:
Done.